contrib/go/src/python/pants/contrib/go/tasks/go_test.py

```diff
@@ -32,9 +33,7 @@ class GoTest(PartitionedTestRunnerTaskMixin, GoWorkspaceTask):
   @classmethod
   def register_options(cls, register):
     super(GoTest, cls).register_options(register)
-    # TODO: turn these into a list of individually-shlexed strings and deprecate using a single
```

Comment:
This is a backwards incompatible change, so to do this you'd need to deprecate the old option and create a new one.

Reply:
Will do!

Reply:
Done!